openmetadata-ui/src/main/resources/ui/src/pages/UserListPage/UserListPageV1.tsx

```diff
@@ -193,7 +205,7 @@ const UserListPageV1 = () => {
     return <Loader />;
   }
 
-  return (
+  return viewPermission ? (
```

Comment:
If no view permission we should remove router is what we decided

Reply:
Yes, I will remove this.

Reply:
Updated!